pkg/tcpip/network/ipv6/ndp_test.go

```diff
@@ -373,13 +373,10 @@ func TestHopLimitValidation(t *testing.T) {
 			invalid := stats.Invalid
 			typStat := typ.statCounter(stats)
 
-			extraDataLen := len(typ.extraData)
-			hdr := buffer.NewPrependable(header.IPv6MinimumSize + typ.size + extraDataLen)
-			extraData := buffer.View(hdr.Prepend(extraDataLen))
-			copy(extraData, typ.extraData)
-			pkt := header.ICMPv6(hdr.Prepend(typ.size))
-			pkt.SetType(typ.typ)
-			pkt.SetChecksum(header.ICMPv6Checksum(pkt, r.LocalAddress, r.RemoteAddress, extraData.ToVectorisedView()))
+			icmp := header.ICMPv6(make(buffer.View, typ.size+len(typ.extraData)))
```

Comment:
Just realized where possible we should not use make(..). Let's make sure we use buffer.NewView() so that later when we want to replace the View implementation its a bit easier. Just noticed this here but I think there are more instances of it in this cl.